### ISSUE TITLE
Script aborts if a docstring contains unhandled fields (found in: 0.1a6)

### ISSUE THREAD

**Issue by UwePfautsch** [UwePfautsch/doxyfilter_python#2]
Example:

````
def foo():
"""
:anytext:
"""
````

leads to the following abort:

````
Traceback (most recent call last):
  File "doxyfilter_python.py", line 263, in <module>
    main(sys.argv[1])
  File "doxyfilter_python.py", line 251, in main
    perform_fh(sys.stdin)
  File "doxyfilter_python.py", line 229, in perform_fh
    print(r'{}{} \param {} ({}) {}'.format(fstr['indent'], comment, param, types[param], text))
KeyError: 'anytext'
````

**Comment by UwePfautsch** [comment]
This behavior also occurs if parameters specified in the docstring are not defined in the function:

````
def doxystr_undefparam():
    """ docstring contains undefined parameters

    :param a: undefined parameter
    """
    pass
````

**Comment by UwePfautsch** [comment]
Issue resolved with version 0.1a7

In case of undefined parameters or unhandled fields a warning-message is written on stdout:

````
WARNING: parameter 'a' unknown at 'def doxystr_undefparam()' is not applied
````
